docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -37,6 +38,80 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.2.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  June 26, 2024
+
+### New Features and Changes
+
+* __GDC 1.0 (Legacy Version)__:
+    * The legacy version (GDC 1.0) of the Data Portal has been officially retired.
+    * GDC 1.0 can no longer be reached.
```

Comment:
@wwysoc2 do we want to highlight this more or something? (P.S. I have no idea what to call it so I hedged.)

Reply:
I vote that we just call it "GDC 1.0" and remove any mention of legacy to avoid confusion ("I know the _legacy_ portal is gone, but what about the regular portal from 2016?")

Reply:
adjusted. changed from 2 lines to 1 cos it started sounding weird. gave it a category despite it just being one bullet so it can legitimately be at the top. :) please check, @wwysoc2